### ISSUE TITLE
Feature request: adding blade absolute position to the OutListParameter

### ISSUE THREAD

**Issue by JelmerPolman** [OpenFAST/openfast#417]
In OpenFAST, plotting aerodynamic values or blade deflections along the undeflected blade radius can get tedious.

For example, plotting the lift from AeroDyn along the blade radius involves:
- reading the chosen blade nodes from the AeroDyn primary input file
- reading the local span from the blade data input file
- reading the sensor output from the OpenFAST output file

Plotting the blade deflection from BeamDyn along the blade radius involves:
- reading the chosen blade nodes from the BeamDyn primary input file
- reading the quadrature nodes from the BeamDyn summary file
- reading the sensor output from the OpenFAST output file

If one wants to plot those sensors along a _deflected_ blade radius, things get more complicated.

A much easier and cleaner workflow would be if the absolute position of the blade aerodynamic and strucutral nodes are added to the OutListParameters. By doing so, one would have to read only the output file. Moreover, plotting against the blade radius is equal to plotting a sensor vs another sensor, which is a simple task for post-processing software.

**Comment by andrew-platt** [comment]
It is currently possible to get the absolute position information from the VTK output.  This data can be visualized using ParaView, or read into a separate post processor.  The drawback with this is that a very large amount of data is created.

I agree that adding the blade position data in absolute coordinates to the standard output file would greatly simplify post-processing, so I'm inclined to support this idea.  @jjonkman, do you have an opinion?

**Comment by jjonkman** [comment]
I agree that it would be useful to add the global blade position outputs.  This would apply to both ElastoDyn blade and BeamDyn outputs (labels now added).  The tower outputs from ElastoDyn are already available in both local and global coordinates.
